OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/Optimove.java

```diff
@@ -69,8 +69,23 @@ final public class Optimove {
 
     private static OptimoveConfig currentConfig;
 
-    private Optimove(@NonNull Context context) {
+    private Optimove(@NonNull Context context, OptimoveConfig config) {
         this.context = context;
+
+        if (!config.isOptimoveConfigured()) {
```

Comment:
If Optimove is not configured, any subsequent calls to public API's like reportEvent will result in a crash.

We should consider whether we need this "non Optimove configured" mode

Reply:
copied from slack

party says any single credentials are enough. you may want to have optimobile without optimove if a) T&T credentials are not ready yet (it may take longer to create them?). This lets you start integrating optimobile earlier b) not sure if optimobile without optimove can exist forever
yes, it would crash. similarly using optimobile method without optimobile credentials would crash. The proposed solution was documenting it properly